code/modules/surgery/surgery_step.dm

```diff
@@ -265,7 +265,7 @@
  */
 /datum/surgery_step/proc/display_pain(mob/living/target, pain_message, mechanical_surgery = FALSE)
 	if(target.stat < UNCONSCIOUS)
-		if(HAS_TRAIT(target, TRAIT_ANALGESIA))
+		if(HAS_TRAIT(target, TRAIT_ANALGESIA) || target.has_trauma_type(/datum/brain_trauma/special/tenacity))
```

Comment:
Wait the trauma gives you `TRAIT_ANALGESIA`? This is redundant I think

Reply:
honk.